mitiq/observable/pauli.py

```diff
@@ -160,17 +163,19 @@ def __repr__(self) -> str:
 
 
 class PauliStringCollection:
+    """A collection of PauliStrings that qubit-wise commute and so can be
+    measured with a single circuit.
+    """
+
     def __init__(
         self, *paulis: PauliString, check_precondition: bool = True
     ) -> None:
-        """Initializes a ``PauliStringCollection``, a collection of
-        ``PauliString``s which qubit-wise commute and so can be measured with
-        a single circuit.
```

Comment:
Why remove?

Reply:
I did not remove it. I added information under the `class` signature, to create a docstring there (if at the top-level there is no docstring, the entire object is not indexed by API-doc). I kept here under the `init` only the part `Initializes a `PauliStringCollection`.`, since other general information is under the `class` signature.